src/ray/util/util.h

```diff
@@ -1,7 +1,9 @@
 #ifndef RAY_UTIL_UTIL_H
 #define RAY_UTIL_UTIL_H
 
+#include <boost/system/error_code.hpp>
```

Comment:
What is this used for?

Reply:
There is an inline function in this header file `inline ray::Status boost_to_ray_status(const boost::system::error_code &error)` which needs this header file. In other places where this file is used, they have this file included outside this header file, so there is no failure.